ansible/provision.yaml

```diff
@@ -35,8 +44,13 @@
     - role: config
     - role: networking
     - role: providers
+    - role: config_containerd
     - role: images
       when: download_images
+    # NOTE(jkoelker) the images task will set the correct `pause_image` to
+    #                reconfigure containerd with.
+    - role: config_containerd
+      when: download_images
```

Comment:
This is kinda a dirty hack. I initially attempted to move the `kubeadm` discovery into the `packages` role, but its already doing *alot* and ordering everything made it more messy.

In a future, refactor we should rethink the separation of the `packages` and `config` roles, and move all the config from the packages into it (so packages can just worry about what packages and from what repos). Then a new role after `config` to "determine" the images needed by `kubeadm` to set the facts, that images will then consume.

Reply:
Hrm... Now that I typed that out, I can just move the `kubeadm` image fetching now into a new role, and run it before the `config` role, and merge the `config_containerd` into it. It wont fix all of the config but it will be a move in the right direction. I'll do that tomorrow night.